src/fpsemi-examples.cpp

```diff
@@ -954,6 +820,111 @@ namespace libsemigroups {
       return p;
     }
 
+    Presentation<word_type> partition_monoid_Eas11(size_t n) {
+      if (n < 4) {
+        LIBSEMIGROUPS_EXCEPTION(
+            "the 1st argument (degree) must be at least 4, found {}", n);
+      }
+
+      Presentation<word_type> p;
+
+      p.alphabet(4);
+      p.contains_empty_word(true);
+
+      // V1
+      presentation::add_rule_no_checks(p, pow(1_w, n), {});
+      presentation::add_rule_no_checks(p, pow(01_w, n - 1), {});
+      presentation::add_rule_no_checks(p, 00_w, {});
+      for (size_t i = 2; i <= n / 2; ++i) {
+        presentation::add_rule_no_checks(
+            p, pow(pow(1_w, i) + 0_w + pow(1_w, n - i) + 0_w, 2), {});
+      }
+
+      // V2
+      presentation::add_rule_no_checks(p, 22_w, 2_w);
+      presentation::add_rule_no_checks(p, 232_w, 2_w);
+      presentation::add_rule_no_checks(p, 012_w + pow(1_w, n - 1) + 0_w, 2_w);
+      presentation::add_rule_no_checks(
+          p, 10_w + pow(1_w, n - 1) + 210_w + pow(1_w, n - 1), 2_w);
+
+      // V3
+      presentation::add_rule_no_checks(p, 33_w, 3_w);
+      presentation::add_rule_no_checks(p, 323_w, 3_w);
+      presentation::add_rule_no_checks(p, 30_w, 3_w);
+      presentation::add_rule_no_checks(p, 03_w, 3_w);
+      presentation::add_rule_no_checks(
+          p, 110_w + pow(1_w, n - 2) + 3110_w + pow(1_w, n - 2), 3_w);
+      presentation::add_rule_no_checks(p,
+                                       pow(1_w, n - 1) + 010_w + pow(1_w, n - 1)
+                                           + 310_w + pow(1_w, n - 1) + 01_w,
+                                       3_w);
+
+      // V4
+      presentation::add_rule_no_checks(p, 0202_w, 202_w);
+      presentation::add_rule_no_checks(p, 2020_w, 202_w);
+
+      // V5
+      presentation::add_rule_no_checks(
+          p, 313_w + pow(1_w, n - 1), 13_w + pow(1_w, n - 1) + 3_w);
+
+      // V6
+      presentation::add_rule_no_checks(
+          p, 3113_w + pow(1_w, n - 2), 113_w + pow(1_w, n - 2) + 3_w);
+      // V7
+      presentation::add_rule_no_checks(
+          p, 3112_w + pow(1_w, n - 2), 112_w + pow(1_w, n - 2) + 3_w);
+      return p;
+    }
+
+    Presentation<word_type> partition_monoid_Machine(size_t n) {
```

Comment:
The presentations provided here don't contain the empty rule. Should it?

Reply:
Yup